pnpm-lock.yaml

```diff
@@ -2840,7 +2837,7 @@ packages:
     peerDependencies:
       react: '*'
     dependencies:
-      '@types/react': 17.0.40
```

Comment:
I will look into why this is being reverted on my machine
cc @mitchellmarkoff

Reply:
I downgraded the `@types/react` in `web` to v17 which should be consistent with what is there now.